concurrent.lib.sh

```diff
@@ -315,6 +315,7 @@ concurrent() (
             while read -r meta; do
                 printf "meta:%d:%s\n" "${1}" "${meta}" >> "${4}"
             done
+            exit ${PIPESTATUS[@]}
```

Comment:
We want the first item (`${PIPESTATUS[0]}`), not all items (`${PIPESTATUS[@]}`).